nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/commands/DeleteJvmOptions.java

```diff
@@ -162,7 +162,8 @@ private void deleteX(final JvmOptionBag bag, final List<String> toRemove, final
         SingleConfigCode<JvmOptionBag> scc = (JvmOptionBag bag1) -> {
             List<String> jvmopts = new ArrayList<>(bag1.getJvmRawOptions());
             int orig = jvmopts.size();
-            boolean removed = jvmopts.removeIf(option -> toRemove.contains(new JvmOption(option).option));
+            // using new JvmOption(option).toString() (instead op option directly) to make sure the correct formatting is applied.
+            boolean removed = jvmopts.removeIf(option -> toRemove.contains(new JvmOption(option).toString()));
```

Comment:
This method has expectations ... it expects that formatting is done in option.toString() and it's result will be same as strings in bag1.getJvmRawOptions(). 
Seems fragile to me, 

getJvmRawOptions does not have javadoc, but it looks it is the content of the XML element.
JvmOption.toString prints also minVersion and maxVersion, seems it is really symetric at this moment ... ok.